internal/tftime/resource_time_offset.go

```diff
@@ -3,6 +3,7 @@ package tftime
 import (
 	"context"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```

Comment:
Looks like imports may need sorting.

If you're using GoLand then you can configure this to happen automatically using the following settings in `Preferences`:

<img width="985" alt="image" src="https://user-images.githubusercontent.com/4332332/183049738-48f6e90f-0d94-452e-a173-dc1fb4171830.png">